modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/JavaClientCodegen.java

```diff
@@ -224,7 +225,12 @@ public void processOpts() {
             apiTemplateFiles.put("apiImpl.mustache", "Impl.java");
             apiTemplateFiles.put("rxApiImpl.mustache", ".java");
             supportingFiles.remove(new SupportingFile("manifest.mustache", projectFolder, "AndroidManifest.xml"));
-        } else {
+        } else if ("jaxrs-spec".equals(getLibrary())) {
+        	// Don't include Jackson and GSON
+        	additionalProperties.remove("jackson");
```

Comment:
Do you need to remove them since they were not added ?

Reply:
@keithchong that should be already done in the latest master: https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/resources/Java/auth/HttpBasicAuth.mustache#L7-L13